GitUI/Translation/English.xlf

```diff
@@ -3050,6 +3046,10 @@ revision:</source>
         <source>Cleanup</source>
         <target />
       </trans-unit>
+      <trans-unit id="buttonClose.Text">
```

Comment:
There is some way to avoid that this text (as well as the other using the new translated text), I do not recall how...

Reply:
Maybe remove the xx.Text="..." in FormCleanupRepository.Designer.cs and move it in the constructor of the form?

Reply:
Should be OK, I believe there is some other trick too.
(This could maybe be fixed in a follow up, there are likely other similar strings.)

Reply:
In in front of my computer, so you have to have a look at another form but to prevent the string to be translated, you should rename the button with the prefix looking link `__no_translate`

Reply:
I just tried, it doesn't work :/

Reply:
The prefix is `_NO_TRANSLATE_`.